tensorboard/plugins/image/tf_image_dashboard/tf-image-dashboard.html

```diff
@@ -222,6 +261,19 @@ <h3>No image data was found.</h3>
         });
       },
 
+      _resetBrightness() {
+        this._brightnessAdjustment = 1;
+      },
+      _resetContrast() {
+        this._contrastPercentage = 100;
```

Comment:
Could we store the 2 default values in a `readOnly` property of the component? ie,

```javascript
_defaultBrightnessAdjustment: {
  type: Number,
  value: 1,
  readOnly: true,
},
```